pkg/ibmvpc/parser.go

```diff
@@ -832,6 +833,12 @@ func getTgwObjects(c *datamodel.ResourcesContainerModel,
 			// TGW's destSubnets contains subnets from its connected VPCs which are contained within routes from its table
 			tgwMap[tgwUID].destSubnets = append(tgwMap[tgwUID].destSubnets, getVPCdestSubnetsByAdvertisedRoutes(tgwMap[tgwUID], vpc)...)
 			tgwMap[tgwUID].addSourceAndDestNodes()
+
+			// explainability related computation
+			// vpcApsPrefixes is a map from the vpc's ap to the index of the prefix that matches it, -1 for default if no match
+			// if getVPCAdvertisedRoutes completed without an error, so would getVpcApsPrefixes
+			vpcApsPrefixes, _ := getVpcApsPrefixes(tgwConn, vpc)
```

Comment:
because, as mentioned here, there is no way `getVpcApsPrefixes` while `getVPCAdvertisedRoutes`  completed successfully. and unless `getVPCAdvertisedRoutes`  completed successfully we would not get here.

Reply:
anyways, after merging the functionality this comment is no longer relevant